samples/CONTRIBUTING.md

```diff
@@ -0,0 +1,85 @@
+# UI-KIT Samples
+
+This guide outlines the conventions and best practices for creating and maintaining samples in the ui-kit repository.
+
+## Overview
+
+The UI-Kit samples are organized into three main entry points that represent the primary ways developers interact with the UI-Kit:
+
+- **`atomic/`** - Samples using Atomic components in every form
+- **`headless/`** - Samples using headless controllers for custom UI implementations  
+- **`headless-ssr/`** - Samples demonstrating server-side rendering with headless controllers
+
+Sample Organization Structure:
+
+```
+samples/
+├── atomic/
+│   └── ...
+├── headless/
+│   └── ...
+└── headless-ssr/
+    └── ...
+```
+
+## Naming Convention
```

Comment:
We could also establish a convention for the name in the package.json

e.g.,

@coveo/headless-ssr-search-nextjs-pages-router-sample

it's a bit of a mouthful though...